sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -400,6 +405,16 @@ multilib_src_install_all() {
 		dosym ../../../lib/systemd/systemd-shutdown /usr/lib/systemd/systemd-shutdown
 	fi
 
+	# workaround for https://github.com/systemd/systemd/issues/13501
+	if use hostnamed-fallback; then
+		# this file requires dbus-broker
+		insinto /usr/share/dbus-1/system.d/
+		doins "${FILESDIR}/org.freedesktop.hostname1_no_polkit.conf"
+
+		insinto "$(systemd_get_systemunitdir)"/systemd-hostnamed.service.d/
```

Comment:
Use `${rootprefix}/lib/systemd/system` in place of `$(systemd_get_systemunitdir)`